code/datums/components/plumbing/_plumbing.dm

```diff
@@ -101,9 +101,14 @@
 		var/datum/component/plumbing/supplier = A
 		if(supplier.can_give(amount, reagent, net))
 			valid_suppliers += supplier
+	// Need to ask for each in turn very carefully, making sure we get the total volume. This is to avoid a division that would always round down and become 0
+	var/targetVolume = src.reagents.total_volume + amount
```

Comment:
src is implicit here, even if it adds clarity, it's generally recommended to not use it when not needed

Reply:
As in I can just reach `reagents.total_volume` without the `src`? I'm brand new to byond/dm/tgstation code.

Reply:
Did just that and tested it locally. Worked.